app/helpers/think_feel_do_dashboard/memberships_helper.rb

```diff
@@ -0,0 +1,10 @@
+module ThinkFeelDoDashboard
+  # Used as an aid to membership creation
+  module MembershipsHelper
+    def projected_end_date(membership)
+      offset = Rails.application.config.study_length_in_weeks * 7
+      start_day = membership.start_date ? membership.start_date : Date.today
+      (start_day + offset.days).strftime("%m/%d/%Y")
```

Comment:
What do you think about moving this format to the i18n config? http://guides.rubyonrails.org/v4.1.8/i18n.html#adding-date-time-formats

This would make it easier to change formats across the engine and customize them in a host app as necessary. Plus it should be a really quick fix.